tdrs-backend/tdpservice/settings/common.py

```diff
@@ -198,30 +200,30 @@ class Common(Configuration):
         "filters": {"require_debug_true": {"()": "django.utils.log.RequireDebugTrue"}},
         "handlers": {
             "django.server": {
-                "level": "INFO",
+                "level": LOGGING_LEVEL,
                 "class": "logging.StreamHandler",
                 "formatter": "django.server",
             },
             "console": {
-                "level": "DEBUG",
+                "level": LOGGING_LEVEL,
                 "class": "logging.StreamHandler",
                 "formatter": "simple",
             },
             "mail_admins": {
-                "level": "ERROR",
+                "level": LOGGING_LEVEL,
```

Comment:
We're not actually using the admin email functionality ([the emails specified in our settings are invalid](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/tdrs-backend/tdpservice/settings/common.py#L90))

However, if we were I'd think we wouldn't want to send `INFO` level logs to those emails. I'd suggest either removing this handler for simplicity or just keeping this one at `ERROR`